js-tests/build/specs.js

```diff
@@ -58,7 +58,7 @@ var Shortcode = require('../../js/src/models/shortcode');
 var InnerContent = require('../../js/src/models/inner-content');
 var ShortcodeAttribute = require('../../js/src/models/shortcode-attribute');
 var ShortcodeAttributes = require('../../js/src/collections/shortcode-attributes');
-var $ = (typeof window !== "undefined" ? window.jQuery : typeof global !== "undefined" ? global.jQuery : null);
+var $ = (typeof window !== "undefined" ? window['jQuery'] : typeof global !== "undefined" ? global['jQuery'] : null);
```

Comment:
@noppanit I don't think so, just tried installing `grunt-browserify` at 3.4 and no diff generated.

Would you mind running locally and seeing if you generate a diff from this branch? If so, maybe you can push it? The change seems innocuous, I just don't understand why it's happening on this branch.